src/org/abymaly/code/calculator/Operation.java

```diff
@@ -0,0 +1,47 @@
+package org.abymaly.code.calculator;
+
+public enum Operation {
+    ADDITION("+") {
+        double calc (double x, double y) {
+            return x + y;
+        }
+    },
+    SUBSTRACTION ("-"){
+        double calc (double x, double y) {
+            return x - y;
+        }
+    },
+    MULTIPLICATION ("*"){
+        double calc (double x, double y) {
+            return x * y;
+        }
+    },
+    DIVISION ("/"){
+        double calc (double x, double y) {
+            return x / y;
+        }
+    },
+    EQUAL ("="){
```

Comment:
Ta implementacja nie zrobi nic, oprócz tego, że zgubi wartość ze stosu. Dlatego, bo jeśli już, to powinien być operator jedno-argumentowy, podobnie jak +/- czy sqrt, ale do tego powinien być osobny enum_operatorow_jedno_argumentowych.